.github/workflows/build.yml

```diff
@@ -10,6 +10,6 @@ jobs:
       - name: Compile Tests
         run: cargo build --verbose
       - name: Unit Tests
-        run: cargo test
+        run: cargo test --lib
       - name: Linter
-        run: cargo clippy --all-targets --all-features -- -D warnings --verbose
+        run: cargo clippy --all-features -- -D warnings --verbose
```

Comment:
Don't forget to modify `.travis.yml` as well

Reply:
https://github.com/bitwyre/websocket_core/pull/2/files#diff-354f30a63fb0907d4ad57269548329e3L15